src/boot/grub2/grub2-15_ostree

```diff
@@ -26,6 +26,16 @@ if ! test -d /ostree/repo; then
     exit 0
 fi
 
+# Gracefully exit if the grub2 configuration has BLS enabled,
+# and the installed version has support for the blscfg module.
+# Since there is no need to create menu entries for that case.
+# See: https://src.fedoraproject.org/rpms/grub2/c/7c2bab5e98d
+. /etc/default/grub
+if test -f /boot/grub2/.grub2-blscfg-supported && \
+   test ${GRUB_ENABLE_BLSCFG} = "true"; then
```

Comment:
```suggestion
   test "${GRUB_ENABLE_BLSCFG}" = "true"; then
```
to avoid a syntax error and crash if the value isn't set.  (Yes, shell script is an awful programming language)

Reply:
> to avoid a syntax error and crash if the value isn't set. (Yes, shell script is an [awful programming language](https://www.oilshell.org/blog/2018/01/28.html))

Indeed, I usually run `shellcheck` to catch these but forgot this time. Pushed that change and also rebased the branch. Thanks a lot.